examples/rest-stickies-csv/app.js

```diff
@@ -32,964 +38,273 @@ app.engine(
 // Configure handlebars
 app.set("view engine", "hbs");
 
-// Declare global variable for Miro API endpoint (App Cards)
-const requestUrl = `https://api.miro.com/v2/boards/${process.env.boardId}/sticky_notes`;
-
-// Declare global access_token variable
-let oauthAccessToken;
-
-let tagData;
-let tagData2;
-let tagData3;
-let tagData4;
-
-// Clean Tag IDs from response
-let tagId;
-let tagId2;
-let tagId3;
-let tagId4;
-
-// Declare variables for connector endpoint ids
-let endpoint1Id;
-let endpoint2Id;
-let endpoint3Id;
-let endpoint4Id;
-let endpoint5Id;
-let endpoint6Id;
-let endpoint7Id;
-let endpoint8Id;
-let endpoint9Id;
-let endpoint10Id;
-let endpoint11Id;
-let endpoint12Id;
-
 // <-------- ROUTES -------->
 
 // ROUTE (GET): Retrieve access_token from OAuth redirect
-app.get("/authorized", (req, res) => {
-  let parsedUrl = req.url;
-  let urlValues = parsedUrl.split("?");
-  console.log(urlValues[1]);
-  oauthAccessToken = urlValues[1];
+app.get("/authorized", async (req, res) => {
+  try {
+    await miro.handleAuthorizationCodeRequest(USER_ID, req);
+  } catch (e) {
+    console.error(e);
```

Comment:
not important for the demo but it is a good practice to render an error page to the user in error handlers